apps/protocol-frontend/src/components/DaoSettingsMembersTable.tsx

```diff
@@ -111,11 +114,61 @@ const DaoSettingsMembersTable = ({ daoId }: DaoSettingsMembersTableProps) => {
         header: 'Member Address',
         accessorKey: 'user',
         cell: ({ getValue }: { getValue: Getter<UIGuildUsers['user']> }) => {
-          return <Text whiteSpace="normal">{getValue().address}</Text>;
+          return getValue().address === userData?.address ? (
+            <Text
+              whiteSpace="normal"
+              fontWeight="bold"
+              bgGradient="linear(to-l, #7928CA, #FF0080)"
+              bgClip="text"
+            >
+              {getValue().address}
```

Comment:
Let's move getValue to a variable to make this DRYer

Reply:
Pretty minor question, but what do we think about destructuring this vs. using a var:

`const { address } = getValue()`
vs. 
`const memberAddress = getValue().address`

Destructuring requires us to use the value as it exists (ie. `address`) whereas the variable lets us create a more expressive name, such as `memberAddress`

Thoughts @alexkeating @amrro

Reply:
I was thinking more `const value = getValue()`. I am find with either.

Reply:
Doing the `const value = getValue()` and using dot notation worked with the typeguard.